connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -416,48 +416,6 @@ public void testCreateConnectorFailedCustomValidation() throws Exception {
     }
 
     @Test
-    public void testConnectorNameConflictsWithWorkerGroupId() throws Exception {
```

Comment:
Besides not removing this, it seems we should also probably have a unit test validating the new behavior for naming consumer groups in sinks?